airflow/providers/ftp/hooks/ftp.py

```diff
@@ -60,8 +60,10 @@ def get_conn(self) -> ftplib.FTP:
         if self.conn is None:
             params = self.get_connection(self.ftp_conn_id)
             pasv = params.extra_dejson.get("passive", True)
+            encoding = params.extra_dejson.get("encoding", "latin-1")
```

Comment:
is there a reason this defaults to latin-1 and not utf-8?

Reply:
yes

Reply:
What is the current behaviour? I am not familiar with FTP, is “server default” a thing? If there is, we should probably do something like

```python
try:
    self.conn.encoding = params.extra_dejson["encoding"]
except KeyError:
    pass
```

Reply:
conn.encoding is default config. I used if to judge whether encoding is in extra_dejson, not try